resources/options_lookup.tsv

```diff
@@ -1422,8 +1450,13 @@ Plug Loads	200%	ResidentialMiscPlugLoads	mult=2.0	"weekday_sch=0.04, 0.037, 0.03
 Misc Freezer	"EF 12, National Average"	ResidentialMiscLargeUncommonLoads	has_freezer=true	freezer_rated_annual_energy=935	freezer_mult=0.342	"freezer_weekday_sch=0.040, 0.039, 0.038, 0.037, 0.036, 0.036, 0.038, 0.040, 0.041, 0.041, 0.040, 0.040, 0.042, 0.042, 0.042, 0.041, 0.044, 0.048, 0.050, 0.048, 0.047, 0.046, 0.044, 0.041"	"freezer_weekend_sch=0.040, 0.039, 0.038, 0.037, 0.036, 0.036, 0.038, 0.040, 0.041, 0.041, 0.040, 0.040, 0.042, 0.042, 0.042, 0.041, 0.044, 0.048, 0.050, 0.048, 0.047, 0.046, 0.044, 0.041"	"freezer_monthly_sch=0.837, 0.835, 1.084, 1.084, 1.084, 1.096, 1.096, 1.096, 1.096, 0.931, 0.925, 0.837"	freezer_location=auto																																									
+Misc Freezer	"EF 16, 2001 Fed Standard (reference freezer)"	ResidentialMiscLargeUncommonLoads	has_freezer=true	freezer_rated_annual_energy=712	freezer_mult=0.342	"freezer_weekday_sch=0.040, 0.039, 0.038, 0.037, 0.036, 0.036, 0.038, 0.040, 0.041, 0.041, 0.040, 0.040, 0.042, 0.042, 0.042, 0.041, 0.044, 0.048, 0.050, 0.048, 0.047, 0.046, 0.044, 0.041"	"freezer_weekend_sch=0.040, 0.039, 0.038, 0.037, 0.036, 0.036, 0.038, 0.040, 0.041, 0.041, 0.040, 0.040, 0.042, 0.042, 0.042, 0.041, 0.044, 0.048, 0.050, 0.048, 0.047, 0.046, 0.044, 0.041"	"freezer_monthly_sch=0.837, 0.835, 1.084, 1.084, 1.084, 1.096, 1.096, 1.096, 1.096, 0.931, 0.925, 0.837"	freezer_location=auto																																									
+Misc Freezer	"EF 18, 2008 Energy Star"	ResidentialMiscLargeUncommonLoads	has_freezer=true	freezer_rated_annual_energy=641	freezer_mult=0.342	"freezer_weekday_sch=0.040, 0.039, 0.038, 0.037, 0.036, 0.036, 0.038, 0.040, 0.041, 0.041, 0.040, 0.040, 0.042, 0.042, 0.042, 0.041, 0.044, 0.048, 0.050, 0.048, 0.047, 0.046, 0.044, 0.041"	"freezer_weekend_sch=0.040, 0.039, 0.038, 0.037, 0.036, 0.036, 0.038, 0.040, 0.041, 0.041, 0.040, 0.040, 0.042, 0.042, 0.042, 0.041, 0.044, 0.048, 0.050, 0.048, 0.047, 0.046, 0.044, 0.041"	"freezer_monthly_sch=0.837, 0.835, 1.084, 1.084, 1.084, 1.096, 1.096, 1.096, 1.096, 0.931, 0.925, 0.837"	freezer_location=auto																																									
+Misc Freezer	"EF 20, 2008 Energy Star Most Efficient"	ResidentialMiscLargeUncommonLoads	has_freezer=true	freezer_rated_annual_energy=568	freezer_mult=0.342	"freezer_weekday_sch=0.040, 0.039, 0.038, 0.037, 0.036, 0.036, 0.038, 0.040, 0.041, 0.041, 0.040, 0.040, 0.042, 0.042, 0.042, 0.041, 0.044, 0.048, 0.050, 0.048, 0.047, 0.046, 0.044, 0.041"	"freezer_weekend_sch=0.040, 0.039, 0.038, 0.037, 0.036, 0.036, 0.038, 0.040, 0.041, 0.041, 0.040, 0.040, 0.042, 0.042, 0.042, 0.041, 0.044, 0.048, 0.050, 0.048, 0.047, 0.046, 0.044, 0.041"	"freezer_monthly_sch=0.837, 0.835, 1.084, 1.084, 1.084, 1.096, 1.096, 1.096, 1.096, 0.931, 0.925, 0.837"	freezer_location=auto																																									

```

Comment:
`freezer_mult=0.342` should be `freezer_mult=`1.0` for these three, assuming you want them to have a "whole freezer" as opposed to a "national average fraction of a freezer".

Reply:
🤦‍♂ Yuuuup thanks.